hydra-cluster/config/protocol-parameters.json

```diff
@@ -368,19 +368,19 @@
     "maxValueSize": 5000,
     "minPoolCost": 0,
     "minUTxOValue": null,
-    "monetaryExpansion": 1.78650067e-3,
-    "poolPledgeInfluence": 0.1,
+    "monetaryExpansion": 0.1,
+    "poolPledgeInfluence": 0,
     "poolRetireMaxEpoch": 18,
     "protocolVersion": {
         "major": 7,
         "minor": 0
     },
-    "stakeAddressDeposit": 400000,
-    "stakePoolDeposit": 500000000,
-    "stakePoolTargetNum": 50,
+    "stakeAddressDeposit": 0,
+    "stakePoolDeposit": 0,
+    "stakePoolTargetNum": 100,
     "treasuryCut": 0.1,
-    "txFeeFixed": 0,
-    "txFeePerByte": 0,
-    "utxoCostPerWord": 34488,
-    "utxoCostPerByte": 4310
+    "txFeeFixed": 155381,
+    "txFeePerByte": 44,
+    "utxoCostPerByte": 4310,
+    "utxoCostPerWord": null
```

Comment:
ditto

Reply:
Why is this `null` and not 0?

Reply:
I can change this to 0 but node actually returns null. 8.1.2

Reply:
🤔